valence-rs/valence
# Minimal stresser implementation

## Description



An implementation of a Minecraft server stresser for testing purposes.

The potential of this pull request is to implement a minimal stresser binary package that would be bound to the local `valence_protocol` package, so it would be always up to date with the latest Valence Minecraft protocol implementation.

The MVP version is going to be able concurrently connect headless clients to a target Minecraft server.

## Test Plan







Steps:
1. Ensure that the connection mode is offline
2. Run `cargo run --example bench_players` or any other example
3. Run `cargo run --package valence_stresser -- --target 127.0.0.1:25565 --count 1000`
4. Monitor the `bench_players` output
 
#### Related

closes #211
